packages/documentation/src/content/docs/integration/playground/overview.mdx

```diff
@@ -37,7 +37,7 @@ This environment will set up a playground where you can use the Rafiki Admin API
 
 :::note
 
-The Mock ASE provided in this repository is intended solely for internal use and demonstration purposes. It is not designed to serve as a reference architecture. If you are looking for a reference implementation of an ASE, please refer to the Test Wallet.
+The Mock ASE provided in this repository is intended solely for internal use and demonstration purposes. It is not designed to serve as a reference architecture. If you are looking for a reference implementation of an ASE, please refer to the [Test Wallet](https://wallet.interledger-test.dev/).
```

Comment:
Use the LinkOut component, which we love so very much, on the Test Wallet link, please.